docs/src/main/asciidoc/amqp.adoc

```diff
@@ -44,194 +56,261 @@ The solution is located in the `amqp-quickstart` {quickstarts-tree-url}/amqp-qui
 
 == Creating the Maven Project
 
-First, we need a new project. Create a new project with the following command:
+First, we need to create two projects: the _producer_ and the _processor_.
+
+To create the _producer_ project, go to https://code.quarkus.io/?a=amqp-quickstart-producer&nc=true&e=resteasy-reactive-jackson&e=smallrye-reactive-messaging-amqp&extension-search=amqp[code.quarkus.io], and click on the "Generate your application" button.
+Then, download the zip file, and unzip it on your file system.
+
+To create the _processor_ project, go to https://code.quarkus.io/?a=amqp-quickstart-processor&nc=true&e=smallrye-reactive-messaging-amqp[code.quarkus.io], and click on the "Generate your application" button.
```

Comment:
Is this really better than the command lines we had before? What worries me a bit is how stable this will be given the evolution of code.quarkus.io.

Reply:
The command lines generate code that we need to remove (tests and co)

Reply:
Even worse, if you don't pick resteasy, you still gets a GreetingResource

Reply:
I think tests generated with the maven plugin were getting in our way. 
I saw that there is a `-DnoCode=true` option for `quarkus-maven-plugin:create`. We can go with that one.

Reply:
-DnoCode=true works, I'm updating to use this.

Reply:
Fixed.